src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -131,4 +170,69 @@ protected void end() {
             }
         };
     }
+
+    public Command drivetoProximity(double inches) {
+        return new SubsystemCommand("Drive Distance", this) {
+
+            //double realDistanceInches = frontLidar.getDistance(true);
+
+            @Override
+            protected void initialize() {
+                drivePidController.setSetpoint(tempestGyro.getAngle());
+                drivePidController.enable();
+                drivePidController.reset();
+            }
+
+            @Override
+            protected void execute() {
+                m_drive.arcadeDrive(Preferences.getInstance().getDouble(RobotMap.Preferences.K_P, 1), angle);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                if (frontLidar.getDistance(true) <= inches) {
+                    return true;
+                } else {
+                    return false;
+                }
+
+            }
+
+            @Override
+            protected void end() {
+                drivePidController.disable();
+            }
+        };
+    }
+
+    public Command turnByDegrees(double degrees) {
+        return new SubsystemCommand("Turn " + degrees, this) {
+            @Override
+            protected void initialize() {
+                tempestGyro.reset();
+                drivePidController.reset();
+                drivePidController.enable();
+                drivePidController.setAbsoluteTolerance(3);
+                drivePidController.setSetpoint(degrees);
+            }
+
+            @Override
+            protected void execute() {
+                SmartDashboard.putNumber("Drive Angle", angle);
+                m_drive.arcadeDrive(0.0, angle);
+
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return drivePidController.onTarget();
+            }
+
+            @Override
+            protected void end() {
+                drivePidController.disable();
+            }
+        };
+    }
+
 }
```

Comment:
Let's add a command that drives the robot for a given amount of time (since we won't have driveDistance using encoder anymore). Should be an example of using a timeout from something in Manipulator